### PR TITLE
Add hotkeys for Well dialog

### PR DESCRIPTION
**Pull request by eos428**
hotkeys 1-6 for each creature tier and M button for max (buy all)
No particular issue that needs to be solved, but nice to have since we already have town screen hotkeys up already

### CODE REVIEW

**Comment by qarkai** [comment]
Could you please add new hotkey to `fheroes2.key`?

**Comment by eos428** [comment]
@qarkai done :) let me know if there's any tabulation issues or something else required

File: src/fheroes2/game/game_hotkeys.cpp [hunk 1]
```diff
@@ -307,6 +309,8 @@ void Game::HotKeysDefaults( void )
     // town screen only
     key_events[EVENT_TOWN_TAVERN] = KEY_r;
     key_events[EVENT_TOWN_JUMP_TO_BUILD_SELECTION] = KEY_b; // also used to build castle, if starting on a village
+
+    key_events[EVENT_WELL_BUY_ALL_CREATURES] = KEY_m;
```
Comment (ihhub) [inline src/fheroes2/game/game_hotkeys.cpp:313]: Why the key was chosen as **M**?

**Review by ihhub — COMMENTED** [review]
Hi @eos428 , I left just one question. Could you please check it?

File: src/fheroes2/game/game_hotkeys.cpp (hunk 1 above)
Comment (eos428) [reply src/fheroes2/game/game_hotkeys.cpp:313]: The button in the well to buy all creatures is using the `max` button currently. For me, it was either
- M for Max
- A for All
- Space for something that's just a bit closer to the left hand side

I went with M for Max at the moment. But I'm open to suggestions if you want it changed :)

**Comment by ihhub** [comment]
@eos428 , thanks a lot for this improvement!

**Pull request merged**